### PR TITLE
Support Procedures- Postgres 11

### PR DESCRIPTION
**Pull request by jalissonmello**
Procedure was introduced in Postgres 11, so this PR contains the initial support. Soon I will submit some tests files.

https://www.postgresql.org/docs/11/sql-createprocedure.html
https://blog.2ndquadrant.com/postgresql-11-server-side-procedures-part-1/

### CODE REVIEW

**Comment by jalissonmello** [comment]
Drop and Add constraint if table exists, issue #252

**Comment by jalissonmello** [comment]
Some other corrections

Error on disable/enable trigger on trigger that dont exists on old schema.

Copy corrections from:
Support Postgresql 12 -Add support for FUNCTION in CREATE TRIGGER #273 alexander-smyslov 
Fix incorrect disable trigger parsing of quoted identifiers. Bugfixes #271 marcus-kempe
fix for null pointer exception #270 marco44
Fix issue where rule parsing was throwing # 269-marcus-kempe
fixed some crashes #257 d1maxa

**Review by jalissonmello — COMMENTED** [review]
PARTITION BY #274

**Comment by jalissonmello** [comment]
Initial suport do identity column. #277

File: README.md [hunk 1]
```diff
@@ -11,6 +11,21 @@ This repo is mainly unmaintained. But if you found a bug and create a pull reque
 
 ## Changelog
 
+### Version 2.6.1
```
Comment (fordfrog) [inline README.md]: as this introduces new features, let's make it 2.7.0

File: pom.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
     <groupId>cz.startnet</groupId>
     <artifactId>apgdiff</artifactId>
     <name>Another PostgreSQL Diff Tool</name>
-    <version>2.6.0-SNAPSHOT</version>
+    <version>2.6.1-SNAPSHOT</version>
```
Comment (fordfrog) [inline pom.xml]: the same here

File: src/main/java/cz/startnet/utils/pgdiff/PgDiffTriggers.java [hunk 3]
```diff
@@ -219,11 +220,13 @@ private PgDiffTriggers() {
     
                 for (final PgTrigger newTrigger : newRelation.getTriggers()) {
                     
+                 if(Objects.nonNull(oldRelation)){   
                    PgTrigger oldTrigger = oldRelation.getTrigger(newTrigger.getName());
                     if ((newTrigger.isDisable() && oldTrigger==null) || 
                            (oldTrigger!=null && oldTrigger.isDisable()!=newTrigger.isDisable())) {
                         list.add(newTrigger);
                     }
+                 }
```
Comment (fordfrog) [inline src/main/java/cz/startnet/utils/pgdiff/PgDiffTriggers.java]: this is just the indentation, but please indent it correctly

File: src/main/java/cz/startnet/utils/pgdiff/PgDiffViews.java [hunk 4]
```diff
@@ -33,7 +34,12 @@ public static void createViews(final PrintWriter writer,
             final PgSchema oldSchema, final PgSchema newSchema,
             final SearchPathHelper searchPathHelper) {
         for (final PgView newView : newSchema.getViews()) {
-            final PgView oldView = oldSchema.getView(newView.getName());
+            PgView oldView = null;
+            
+            if(Objects.nonNull(oldSchema)){
+               oldView=oldSchema.getView(newView.getName());
+             }
```
Comment (fordfrog) [inline src/main/java/cz/startnet/utils/pgdiff/PgDiffViews.java]: here the indentation too

File: src/main/java/cz/startnet/utils/pgdiff/parsers/CreateIndexParser.java [hunk 5]
```diff
@@ -31,8 +31,9 @@ public static void parse(final PgDatabase database,
         final boolean unique = parser.expectOptional("UNIQUE");
 
         parser.expect("INDEX");
-        parser.expectOptional("CONCURRENTLY");
-
+         if (parser.expectOptional("CONCURRENTLY"));
```
Comment (fordfrog) [inline src/main/java/cz/startnet/utils/pgdiff/parsers/CreateIndexParser.java]: what's the purpose of the ``if`` here?

File: src/main/java/cz/startnet/utils/pgdiff/parsers/CreateTriggerParser.java [hunk 6]
```diff
@@ -108,14 +108,13 @@ public static void parse(final PgDatabase database,
         }
 
         parser.expect("EXECUTE");
-
-        if (parser.expectOptional("PROCEDURE")) {
-            trigger.setFunction(parser.getRest());
-        } else if (parser.expectOptional("FUNCTION")) {
-            trigger.setFunction(parser.getRest());
+        if (parser.expectOptional("PROCEDURE")) {           
+        } else if (parser.expectOptional("FUNCTION")) {
```
Comment (fordfrog) [inline src/main/java/cz/startnet/utils/pgdiff/parsers/CreateTriggerParser.java]: this looks really weird. please use something like ``if (!parser.expectOptional("PROCEDURE") && !parser.expectOptional("FUNCTION")} {...``

File: src/main/java/cz/startnet/utils/pgdiff/parsers/Parser.java [hunk 7]
```diff
@@ -249,7 +254,7 @@ public int parseInteger() {
      *
      * @return parsed string, if quoted then including quotes
      */
-    public String parseString() {
+        public String parseString() {
```
Comment (fordfrog) [inline src/main/java/cz/startnet/utils/pgdiff/parsers/Parser.java]: the alignment here seems to be wrong...

File: src/main/java/cz/startnet/utils/pgdiff/schema/PgTable.java [hunk 8]
```diff
@@ -72,6 +72,12 @@
      * PgSchema
      */
     private final PgSchema schema;
+    
+    /**
+     * Partion By 
+     */
+
```
Comment (fordfrog) [inline src/main/java/cz/startnet/utils/pgdiff/schema/PgTable.java]: no need for extra line

**Comment by jalissonmello** [comment]
Hi.
 If possible, please check the changes again.

File: src/main/java/cz/startnet/utils/pgdiff/schema/PgTable.java (hunk 8 above)
Comment (jalissonmello) [reply src/main/java/cz/startnet/utils/pgdiff/schema/PgTable.java]: Please check the changes again.

**Comment by fordfrog** [comment]
thank you :-)

**Pull request merged**